src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,26 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  inputText: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, inputText }) => {
+  const validMovie = (movie: Movie) => {
+    if (movie.title.toUpperCase().includes(inputText.toUpperCase())
+      || movie.description.toUpperCase().includes(inputText.toUpperCase())) {
+      return true;
+    }
+
+    return false;
+  };
+
+  return (
+    <div className="movies">
+      {movies.map(movie => (
+        validMovie(movie)
+          ? <MovieCard key={movie.imdbId} movie={movie} />
+          : null
```

Comment:
I'd make this validation before I render movies, like applying `filter`

Reply:
Agree, just filter your `movies` with this `isMovieFit` callback before working with them in jsx